webpack/components/extensions/Hosts/BulkActions/BulkPackagesWizard/04_ReviewFooter.js

```diff
@@ -72,10 +86,14 @@ export const BulkPackagesReviewFooter = () => {
     if (selectedAction === INSTALL) {
       triggerBulkPackageInstall();
     }
+    if (selectedAction === REMOVE) {
+      triggerBulkPackageRemove();
+    }
     closeModal();
   };
 
-  const isBulkActionInProgress = isBulkUpgradeInProgress || isBulkInstallInProgress;
+  /* eslint-disable max-len */
+  const isBulkActionInProgress = isBulkUpgradeInProgress || isBulkInstallInProgress || isBulkRemoveInProgress;
```

Comment:
You can just put them on the next line to avoid disabling the rule:
```suggestion
  const isBulkActionInProgress = isBulkUpgradeInProgress ||
    isBulkInstallInProgress || isBulkRemoveInProgress;
```